### DISCUSSION TITLE
vsync doesn't limit to monitor refresh rate

### DISCUSSION THREAD

**Discussion by GDcheerios** [ppy/osu#19434] [Q&A] [answered]
version: **2022.723.0**

The frame rate isn't capped at my monitors refresh rate when I have vsync frame limiter enabled.

I have a 144hz monitor and the vsync frame limiter caps at around 577 while 2x frame limiter caps at 288.
[performance-update.log](https://github.com/ppy/osu/files/9206423/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/9206424/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9206425/updater.log)
[database.log](https://github.com/ppy/osu/files/9206427/database.log)
[network.log](https://github.com/ppy/osu/files/9206428/network.log)
[performance.log](https://github.com/ppy/osu/files/9206430/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/9206431/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/9206432/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/9206433/performance-input.log)

**Comment by smoogipoo** [comment, marked as answer]
Are you looking at the update fps? This is intentional.

**Reply by GDcheerios** [reply]
no, this is both in the new fps and in the more detailed old fps.

so it shows in update and in draw.

**Reply by smoogipoo** [reply]
Can you show a picture?

**Reply by peppy** [reply]
The other possibility is that you have vsync forced off at a driver level.

**Reply by GDcheerios** [reply]
for some reason I can't upload images to github, but I can send a google drive link.

https://drive.google.com/file/d/1VxaHpBTG77nhW4qLkXYVrxmxGYYGbMj8/view?usp=sharing

**Reply by GDcheerios** [reply]
> The other possibility is that you have vsync forced off at a driver level.

ah, yeah this was it. I never knew this was a thing. Thanks @peppy !
